homeassistant/components/template/switch.py

```diff
@@ -138,14 +138,17 @@ async def async_added_to_hass(self) -> None:
             await super().async_added_to_hass()
             if state := await self.async_get_last_state():
                 self._state = state.state == STATE_ON
+        await super().async_added_to_hass()
```

Comment:
We already call this on line 138. We shouldn't do it again in that case, right?

Reply:
You're right, but since this is an old issue I'd prefer to fix it in a separate PR.